html-forms/index.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <title>html forms</title>
+</head>
+<body>
+  <h1>Coding Form!</h1>
+  <form action="" method="GET">
+<label for="email">
+  <div>
+    Email
+  </div>
+</label>
+  <input type="email" name="email" id="email" placeholder="student@lfz.com">
+<label for="reasons">
+  <div>
+    Why is coding cool?
+  </div>
+</label>
+<select name="reasons" id="reasons">
+  <option value="">Select an option</option>
+  <option value="fun">because its fun</option>
+  <option value="powerful">because its powerful</option>
+  <option value="interesting">because its interesting</option>
+</select>
+<label for="languages">
+  <div>
+    Favorite languages (choose one)
+  </div>
+</label>
+
+<input type="radio" name="languages" id="html">
+<label for="html">HTML</label>
+
+<input type="radio" name="languages" id="css">
+<label for="css">CSS </label>
+
+<input type="radio" name="languages" id="js">
+<label for="js">JavaScript </label>
+<div>
+<div>Would you like to sign up for more information?</div>
+<input type="checkbox">
+<label for="checkbox">Sign me up!</label>
+</div>
+<div>
+  <label for="text">Comments</label>
+  <div>
+  <input type="text">
```

Comment:
this should be a textarea input

Reply:
![Screenshot (319)](https://user-images.githubusercontent.com/51901536/135558960-046be050-ac2b-433d-a08f-eb04bd9d37ce.png)